### DISCUSSION TITLE
Which version should I upgrade to?

### DISCUSSION THREAD

**Discussion by coldwinds** [sandboxie-plus/Sandboxie#3988] [Q&A Feedback] [answered]
Hello, everyone.
I'm using a relatively old version(Sandboxie-Classic-x64-v5.53.3) https://github.com/sandboxie-plus/Sandboxie/releases?page=11
Which version should I install for a smooth upgrade?
Or do I just need to install the latest version directly?

**Comment by edions** [comment, marked as answer]
It's always a good idea to go for the latest version, I think you can upgrade to the latest one without any problem . But in case it fails you can just reinstall it just make sure to retain the sandboxie.ini file when prompted by the uninstaller and everything should work smoothly.

**Reply by coldwinds** [reply]
I'll give it a try. Thanks for the information.
